### DISCUSSION TITLE
I can't clone wiki

### DISCUSSION THREAD

**Discussion by fabiohcnobre** [community/community#53548] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Bug

### Body

Because I have an error when cloning a wiki.

![image](https://user-images.githubusercontent.com/22354660/233850091-f9ae115e-8734-428d-8746-e41c545b0dd3.png)

**Comment by fabiohcnobre** [comment]
https://github.com/orgs/community/discussions/53566

**Comment by tuves** [comment]
Hi @fabiohcnobre,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).

**Reply by fabiohcnobre** [reply]
Will this comment stay active? If so, could you switch from Product Feedback to Product Bug.

Because I have an error when cloning a wiki.
